packages/fake-driver/test/e2e/general-tests.js

```diff
@@ -106,6 +106,45 @@ function generalTests() {
       res[0].type.should.eql('pointer');
       res[0].actions.should.have.length(2);
     });
+
+    it('should get and set a fake thing via execute overloads', async function () {
+      let thing = await driver.executeScript('fake: getThing', []);
+      should.not.exist(thing);
+      await driver.executeScript('fake: setThing', [{thing: 1234}]);
```

Comment:
Just a question.
Do we allow `fake:setThing` like current mobile commands?, or will this strictly check the format? I saw some users did not set spaces in past, so I wondered spaces. If we make the format strict, we may need to address as a breaking change in a migration guide (or do something in drivers' update)

Reply:
Yes, this change does not do anything special with spaces. A way to make it a non-breaking change in the driver would be to define the method metadata and then include both ways of accessing, for example:

```js
const SET_THING_CMD = {
  command: 'setThing',
  params: {required: ['thing'], optional: []},
};

static executeMethodMap = {
  'fake: setThing': SET_THING_CMD,
  'fake:setThing': SET_THING_CMD,
}
```

which is slightly more verbose but would keep it from being a breaking change if we want.

or, we could add custom logic into `executeMethods` that looks for both versions if it finds a colon in the command. that seemed too unnecessary for this PR but we could do it.

Reply:
I see, it makes sense as this base driver 🆗  Drivers may need to consider the space thing.